index.html

```diff
@@ -246,6 +246,60 @@ <h2>
           };
         </pre>
     </section>
+    <section>
+      <h2>
+        Extensions to the <a>document</a> interface
+      </h2>
+      <section>
+        <h2>
+          Internal Slots
+        </h2>
+        <table>
+          <thead>
+            <tr>
+              <th>
+                Internal Slot
+              </th>
+              <th>
+                Description (non-normative)
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <dfn data-lt="orientation lock">[[\orientationLock]]</dfn>
+              </td>
+              <td>
+                A <a>document</a>'s orientation lock represented by an
+                unordered set of <a>orientationType</a>
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <dfn data-lt=
+                "default orientation">[[\defaultOrientation]]</dfn>
+              </td>
+              <td>
+                The set of orientations to which the screen orientation is
+                locked when not explicitly locked by this API or any other
+                means.
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <dfn data-lt=
+                "promiseToLockScreen">[[\promiseToLockScreen]]</dfn>
```

Comment:
Remove the `data-lt` here. And always reference the internal slot like this:

```HTML
<a>[[\promiseToLockScreen]]</a>
```

Same with <a>defaultOrientation</a> 

Remember, you can define the **concept** of "default orientation", which is represented by [[\defaultOrientation]].